File: CodeForces/556A-Case of Zeros and Ones/556a.java
import java.util.*;

class Main{

    public static void main(String[] args){
	Scanner sc = new Scanner(System.in);
	int cantidad = sc.nextInt();
	String cadena = sc.nextLine();
	int zero = 0, one = 0;

	for(int i = 0; i < cadena.length(); ++i){
	    if(cadena.charAt(i) == '0') ++zero;
	    else ++one;
	}

	System.out.println(Math.abs(zero - one));

	sc.close();

    }
}

File: CodeForces/556A-Case of Zeros and Ones/556a.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
  int cantidad, one = 0, zero = 0;
  string cadena;
  cin >> cantidad;
  cin >> cadena;

  for(int i = 0; i < cadena.length(); ++i){
    if(cadena[i] == '0') ++one;
    else ++zero;
  }

  cout << abs(zero - one);

  return 0;
}

